packages/react-slider/src/components/Slider/useSliderState.tsx

```diff
@@ -97,9 +107,13 @@ export const useSliderState = (state: SliderState) => {
 
   const onPointerUp = React.useCallback(
     (ev: React.PointerEvent<HTMLDivElement>): void => {
-      disposables.current.forEach(dispose => dispose());
-      disposables.current = [];
       showStepAnimation();
+      setEventData(prevState => ({
+        ...prevState,
+        disabled: true,
+        element: ev.target as HTMLElement,
+        pointerId: ev.pointerId,
+      }));
```

Comment:
Since you are overriding everything in the event data either way shouldn't it be the same to just do the following?
```suggestion
      setEventData({
        disabled: true,
        element: ev.target as HTMLElement,
        pointerId: ev.pointerId,
      });
```